packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -420,6 +436,10 @@ abstract final class SystemChrome {
   ///
   /// ### Android
   ///
+  /// If your Flutter app is targeting Android 16 (API 36) or later and you are using
+  /// a device with a display width >= 600 dp, then you cannot set the device
+  /// orientation. For more details see Android 16 docs [here](https://developer.android.com/about/versions/16/behavior-changes-16#ignore-orientation).
```

Comment:
Could you note the `setPreferredOrientations` method here for clarity? Something like "...you cannot set the device orientation via `setPreferredOrientations`".